starters/vue3-xstate-css/src/router/index.ts

```diff
@@ -1,13 +1,12 @@
 import { createRouter, createWebHistory } from 'vue-router';
-import HomeView from '../views/HomeView.vue';
 
 const router = createRouter({
   history: createWebHistory(import.meta.env.BASE_URL),
   routes: [
     {
       path: '/',
       name: 'home',
-      component: HomeView,
+      component: () => import('../views/HomeView.vue'),
```

Comment:
Made this change to match the other routes and lazy load all the components. The Vue Router docs actually recommend this as well where it's possible and I've verified this works as expected.
Relevant doc page: https://router.vuejs.org/guide/advanced/lazy-loading.html